File: APAC3BrunoLuisVazquezPais/app/src/main/java/com/ieseljust/apac3brunoluisvazquezpais/db/Incidencia.java
package com.ieseljust.apac3brunoluisvazquezpais.model_b;


import androidx.room.Entity;
import androidx.room.PrimaryKey;
import  java.io.Serializable;

@Entity
data class Incidencia(
    @PrimaryKey(autoGenerate = true)
    var id:Int,
    var assumpte: String,
    var descripcio: String,
    var ubicacio: String,
    var servei: String, // Jardineria, Infraestructures, Obres
    var img: Int,
    var resolt: Boolean
): Serializable

File: APAC3BrunoLuisVazquezPais/app/src/main/java/com/ieseljust/apac3brunoluisvazquezpais/model/db/Incidencia.kt
package com.ieseljust.apac3brunoluisvazquezpais.model.db

import java.io.Serializable
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class Incidencia(
    @PrimaryKey(autoGenerate = true)var id:Long = 0,
    var nom:String,
    var descripcio: String?,
    var img: Int,
    var data: String?,
    var hora:String?
): Serializable
